src/extensions/unitstable/Units/TableWidget.tsx

```diff
@@ -139,56 +161,62 @@ const TableWidget: FunctionComponent<Props> = (props) => {
     const primarySortColumnDirection = primaryRule ? (primaryRule.sortAscending ? 'ascending' : 'descending') : undefined
     
     return (
-        <Table className="TableWidget">
-            <HeaderRow
-                columns={columns}
-                primarySortColumnName={primarySortColumnName}
-                primarySortColumnDirection={primarySortColumnDirection}
-                onColumnClick={(column) => {
-                    const columnName = column.columnName
-                    let newSortFieldOrder = [...sortFieldOrder]
-                    if (sortFieldOrder[sortFieldOrder.length - 1] === columnName) {
-                        if (sortFieldOrder[sortFieldOrder.length - 2] === columnName) {
-                            // the last two match this column, let's just remove the last one
-                            newSortFieldOrder = newSortFieldOrder.slice(0, newSortFieldOrder.length - 1)
+        <TableContainer style={height !== undefined ? {maxHeight: height} : {}}>
+            <Table stickyHeader className="TableWidget">
```

Comment:
The top (header) row is now sticky. In order to achieve this, we need to use a TableContainer and set the height explicitly.